### DISCUSSION TITLE
re use twig code

### DISCUSSION THREAD

**Discussion by Elmatbaisaad** [symfony/symfony#46639] [Q&A] [answered]
Hello everyone ,
i want to re use a twig code in other twig 
i tried to do so :
**the main twig:**
![image](https://user-images.githubusercontent.com/101296742/173068542-a93bf78e-e16a-4941-9006-6645d765b26e.png)
**where i want to re use the code:**
![image](https://user-images.githubusercontent.com/101296742/173068675-8ac15518-0d07-4da6-b1aa-7055f843b702.png)
but i keep getting this error:
![image](https://user-images.githubusercontent.com/101296742/173068801-e60add5d-6402-4d57-8add-9d0a71fcf424.png)

thank you !

**Comment by GromNaN** [comment, marked as answer]
The path you used in the `template` tag is incorrect. It cannot be relative to the current file.
I think you should do 
```twig
{{ include('readregistersandcoils/testinclude.html') }}
```

**Reply by Elmatbaisaad** [reply]
thank you i didn't notice it
